src/analysis/validation_with_arguments.Rmd

```diff
@@ -267,31 +295,38 @@ We divide the validation data into classes with respect to the length of the pro
 ```{r}
 validation_long = validation[which(validation$program_length > 400), ]
 validation_short = validation[which(validation$program_length <= 100), ]
+validation_no_exp = validation[which(validation$dominant != 'EXP'), ]
```

Comment:
Why `no_exp`?

Reply:
just for comparison, exp is a very expensive program and including it "fixes" the model in favor of the trivial model. We would like to ensure that even without exp we still improve fit compared to the trivial model (true).